src/pages/taskQueue/TaskQueueTable.tsx

```diff
@@ -5,54 +5,38 @@ import Badge from "@leafygreen-ui/badge";
 import { Body, Disclaimer } from "@leafygreen-ui/typography";
 import { Table } from "antd";
 import { ColumnProps } from "antd/es/table";
-import { useParams, useLocation } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { useTaskQueueAnalytics } from "analytics";
 import { StyledRouterLink, WordBreak } from "components/styles";
-import { getVersionRoute, getTaskRoute } from "constants/routes";
+import {
+  getVersionRoute,
+  getTaskRoute,
+  getUserPatchesRoute,
+} from "constants/routes";
 import {
   DistroTaskQueueQuery,
   DistroTaskQueueQueryVariables,
   TaskQueueItem,
   TaskQueueItemType,
 } from "gql/generated/types";
 import { DISTRO_TASK_QUEUE } from "gql/queries";
-import { usePrevious } from "hooks";
-import { string } from "utils";
 import { formatZeroIndexForDisplay } from "utils/numbers";
-
-const { msToDuration } = string;
+import { msToDuration } from "utils/string";
 
 export const TaskQueueTable = () => {
   const taskQueueAnalytics = useTaskQueueAnalytics();
 
   const { distro, taskId } = useParams<{ distro: string; taskId?: string }>();
 
-  const {
-    data: taskQueueItemsData,
-    loading,
-    refetch: refetchQueue,
-  } = useQuery<DistroTaskQueueQuery, DistroTaskQueueQueryVariables>(
-    DISTRO_TASK_QUEUE,
-    {
-      variables: { distroId: distro },
-      errorPolicy: "ignore",
-      fetchPolicy: "cache-and-network",
-    }
-  );
+  const { data: taskQueueItemsData, loading } = useQuery<
+    DistroTaskQueueQuery,
+    DistroTaskQueueQueryVariables
+  >(DISTRO_TASK_QUEUE, {
+    variables: { distroId: distro },
+  });
 
   const taskQueueItems = taskQueueItemsData?.distroTaskQueue ?? [];
 
-  // REFETCH QUEUE IF URL PARAM CHANGES
```

Comment:
This is the default behavior of useQuery so I'm not sure why this code was here.